timescale/timescale.go

```diff
@@ -146,6 +146,20 @@ func (r repo) RetrieveDistinctIPsCountries(ctx context.Context, filters callhome
 	for _, country := range summary.Countries {
 		summary.TotalDeployments += country.NoDeployments
 	}
+
+	q1 := `select city from telemetry;`
```

Comment:
should be sth like `select distinct city from telemetry`, since if a city has more than one instance of Mainflux it's shown twice as shown below with `kikuyu`
![image](https://github.com/mainflux/callhome/assets/100555904/fbc1fdc7-6ff7-45b0-b271-877e3d099cf2)